go/mysql/fakesqldb/server.go

```diff
@@ -290,6 +293,20 @@ func (db *DB) ConnectionClosed(c *mysql.Conn) {
 
 // ComQuery is part of the mysql.Handler interface.
 func (db *DB) ComQuery(c *mysql.Conn, q []byte, callback func(*sqltypes.Result) error) error {
+	err := db.comQueryImpl(c, q, func(qr *sqltypes.Result) error {
```

Comment:
One problem you'll have to think about is results that lead to other results. Specifically, how to make lookup vindexes return meaningful values so the executor follows through on the routing.